backend/data/final_data/conditionsProcessed.json

```diff
@@ -8490,33 +8490,33 @@
     },
     "MGMT4101": {
         "original": "MGMT2102 and completion of 72 UOC (if you are enrolled the Commerce International [program 3558], completion of the Commerce Overseas Program (Exchange) is also required)<br/><br/>",
-        "processed": "4501",
-        "handbook_note": "enrolment in the International Business Honours plan is required"
+        "processed": "MGMT2102 && 72UOC && 3558",
+        "handbook_note": "Commerce Overseas Program (Exchange) is also required"
```

Comment:
Why was a manual fix done for these that doesn't seem to match the original requirement?
- https://www.handbook.unsw.edu.au/undergraduate/courses/2022/MGMT4101/
- https://www.handbook.unsw.edu.au/undergraduate/courses/2022/MGMT4104/